edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_variables.scss

```diff
@@ -2,14 +2,19 @@
 
 //color
 $primary: #B70005;
+$primary-dark: #a52238;
 $inverse: #FFF;
 $secondary: #E89D00;
 $footer-background: #242E33;
 $footer-contrast: #F2F2F2;
 $dark: #313131;
 $gray: #242E33;
 $light-gray: #D4D7D9;
+$lighter-gray: #dbdbdb;
 $primary-light: #EE3F40;
+$dashboard-contrast: #fafafa;
+$dashboard-course-info-text: #444444;
+$checkbox-border: #999999;
```

Comment:
Is this colour from the mockups? @kuipumu

Reply:
Since this is only used in the dashboard styles, please move it to the dashboard definitions. @kuipumu